src/content/docs/de/guides/imports.mdx

```diff
@@ -0,0 +1,267 @@
+---
+title: Statische Inhalte
```

Comment:
In English the title is just “Imports” which highlights the action of importing rather than specifically the kind of thing you might import. Maybe “Imports” or “Importieren” can work in German too?